### PR TITLE
Fix typo in `Overview`

### PR DESCRIPTION
**Pull request by na2axl**
Before: Overvieew
After: Overview

### CODE REVIEW

**Review by micronAMZN — CHANGES_REQUESTED** [review]
This commit needs to be amended with a sign-off so it passes DCO before it can be merged.

**Comment by na2axl** [comment]
@micronAMZN sorry for that, it should be OK now 🙂

**Review by micronAMZN — APPROVED** [review]

**Pull request merged**
